CHANGELOG.md

```diff
@@ -49,6 +51,8 @@ reconfigure their proxies. Please read the Breaking Changes below thoroughly.
 
 ## Changes since v3.2.0
 
+- [#178](https://github.com/pusher/outh2_proxy/pull/178) Add Silence Ping Logging and Exclude Logging Paths flags (@kskewes)
```

Comment:
This looks like a merge error, these two entries should not be here

Reply:
I have fixed this in order to get it merged.